examples/css/index.html

```diff
@@ -26,22 +26,152 @@ <h1 class="mistica-text-6">Hello mistica css</h1>
-                <div class="tag">Headline</div>
-                <h2 class="mistica-text-cardTitle">Title</h2>
-                <p class="mistica-text-2">Subtitle</p>
-                <p class="description mistica-text-2">Description</p>
-                <button class="button small">Action</button>
-            </section>
+            <div class="components">
+                <h2 class="mistica-text-title3">Components</h2>
+
+                <h3 class="mistica-text-title2">Texts / Titles</h3>
+                <div class="cards-group">
+                    <div>
+                        <p class="mistica-text-1">Text 1</p>
+                        <p class="mistica-text-2">Text 2</p>
+                        <p class="mistica-text-3">Text 3</p>
+                        <p class="mistica-text-4">Text 4</p>
+                        <p class="mistica-text-5">Text 5</p>
+                        <p class="mistica-text-6">Text 6</p>
+                        <p class="mistica-text-7">Text 7</p>
+                        <p class="mistica-text-8">Text 8</p>
+                        <p class="mistica-text-9">Text 9</p>
+                        <p class="mistica-text-10">Text 10</p>
+                    </div>
+                    <div>
+                        <h4 class="mistica-text-title1">Title 1</h4>
+                        <h3 class="mistica-text-title2">Title 2</h3>
+                        <h2 class="mistica-text-title3">Title 3</h2>
+                        <h1 class="mistica-text-title4">Title 4</h1>
+                    </div>
+                </div>
+
+                <h3 class="mistica-text-title2">Tag</h3>
+                <div>
+                    <div class="mistica-tag-promo">Promo</div>
+                    <div class="mistica-tag-active">Active</div>
+                    <div class="mistica-tag-inactive">Inactive</div>
+                    <div class="mistica-tag-success">Success</div>
+                    <div class="mistica-tag-warning">Warning</div>
+                    <div class="mistica-tag-error">Error</div>
+                </div>
+
+                <h3 class="mistica-text-title2">Buttons</h3>
+                <div>
+                    <a class="mistica-button-primary" href="#">Primary</a>
+                    <a class="mistica-button-secondary" href="#">Secondary</a>
+                    <a class="mistica-button-danger" href="#">Danger</a>
+                    <a class="mistica-button-link" href="#">Link</a>
+                    <a class="mistica-button-primary-small" href="#">Primary</a>
+                    <a class="mistica-button-secondary-small" href="#">Secondary</a>
+                    <a class="mistica-button-danger-small" href="#">Danger</a>
+                    <a class="mistica-button-link-small" href="#">Link</a>
+                </div>
+
+                <h3 class="mistica-text-title2">Cards</h3>
+                <h4 class="mistica-text-title1">MediaCard / DataCard / SnapCard</h4>
```

Comment:
- buttonGroup has strange behavior when the buttons dont fit in the container in DataCard
- buttonLink is not bleeding when fall to another line (I suppose that this will be impossible to fix)
![image](https://github.com/user-attachments/assets/6b6dc512-8cb3-4e9b-ae50-af18e303df38)

Reply:
Fixed the second card case. The bleeding case is not easy to fix without js